store/postgres/tests/store.rs

```diff
@@ -92,7 +94,11 @@ where
     runtime
         .block_on(future::lazy(move || {
             // Set up Store
-            let logger = Logger::root(slog::Discard, o!());
+            let logger = match env::var_os("GRAPH_LOG") {
```

Comment:
I'm going to do a proper review later, but this is a great idea. I've often been missing logs from tests due to the `slog::Discard` everywhere.